### DISCUSSION TITLE
Layouts RFC Is scheduled to be release in which Next.js version?

### DISCUSSION THREAD

**Discussion by FelipeBattistotti** [vercel/next.js#40962] [Help] [answered]
### Verify canary release

- [X] I verified that the issue exists in the latest Next.js canary release

### Provide environment information

Next.js version = latest

### What browser are you using? (if relevant)

Chrome 105.0.5195.127

### How are you deploying your application? (if relevant)

Vercel

### Describe the Bug

Layouts RFC [#37136](https://github.com/vercel/next.js/discussions/37136) Is scheduled to be release in which Next.js version?
It's possible to test Layouts now?

### Expected Behavior

Layouts RFC [#37136](https://github.com/vercel/next.js/discussions/37136) Is scheduled to be release in which Next.js version?
It's possible to test Layouts now?

### Link to reproduction

https://github.com/FelipeBattistotti/devinfo-card-frontend.git

### To Reproduce

Layouts RFC [#37136](https://github.com/vercel/next.js/discussions/37136) Is scheduled to be release in which Next.js version?
It's possible to test Layouts now?

**Comment by icyJoseph** [comment]
https://github.com/vercel/next.js/discussions/37136#discussioncomment-3708199

> keep an eye on https://nextjs.org/conf 🙌

**Reply by FelipeBattistotti** [reply]
Thanks @icyJoseph 👍

**Comment by leerob** [comment, marked as answer]
**Update:** We’re excited to announce [Next.js 13](https://nextjs.org/13), which includes the new `app/` directory (beta) with support for nested layouts, colocation of data fetching, streaming, Server Components by default, and much more.

Inside the `app/` directory, there is a powerful new way to fetch data with React’s `use()` hook and the extended Web `fetch()` API. This allows you to colocate data fetching directly inside components for the most flexibility, including inside layouts.

You can leave feedback here: https://github.com/vercel/next.js/discussions/41745
